gulpfile.js

```diff
@@ -402,7 +397,8 @@
 
 			let config = {
 				manifest: cfg.serverConfig,
-				chromiumFlags: JSON.stringify(cfg.chromiumFlags),
+				onElectronClose: (exitCode)=> process.exit(exitCode),
```

Comment:
```suggestion
				onElectronClose: process.exit,
```
I think that could work as well